docs/source/concept_models/vivarium_swissre_lungcancer/concept_model.rst

```diff
@@ -281,42 +333,126 @@ Lung cancers may be detected in one of two ways in this simulation: either via s
 
-5.3.2.3 Screening Sensitivity and Specificity
+5.3.4.3 Screening Sensitivity and Specificity
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Lung cancer screening specificity is assumed to be 100%; in other words, we assume that there will be no false negative lung cancer results detected via screening. 
 
-.. todo::
+Lung cancer screening sensitivity is assumed to be 98.1% (95% CI: 88.4, 99.9); in other words, we assume that the probability that LDCT screening will detect lung cancer given that the individual has lung cancer is 98.1% (95% CI: 88.4, 99.9). This value was obtained from the Yang et al. (2018) lung cancer screening trial conducted in China to be most representative of the model population.
+
+Since the confidence interval for this parameter is not symmetric about the mean, we will assume a lognormal distribution of uncertainty, detailed by the code below. Note that this code plots the inverse of the true distribution and then converts back to a sensitivity parameter. The value sampled by this code will represent sensitivity as a *proportion* of lung cancers successfully detected by LDCT. The sensitivity parameter should be sampled at the *draw* level.
+
+.. code-block:: python
 
-  Document screening sensitivity value and references
+  from scipy.stats import lognorm
 
-5.3.2.4 Screening Coverage
+  # set distribution parameters
+  s = 1
+  loc = 0
+  scale = 1.9
+
+  # sample sensitivity value as PROPORTION of lung cancers successfully detected by LDCT
+  sensitivity = (100 - lognorm.rvs(s, loc, scale, size=1)) / 100
+
+5.3.4.4 Screening Coverage
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Baseline
+To view a write-up of the research background on lung cancer screening coverage in China, download :download:`this word document <screening_coverage_research_background.docx>`. 
 
-.. todo:: 
-  
-  Document values for 20 year lag from US coverage, as suggested by SwissRe. Sex-specific values if possible. 
+The population-level coverage rates for *both* the primary and secondary target screening populations are defined below for both scenarios. Note that the figure represents the population coverage rate, although we will model differences in the sex-specific coverage rates, as described below. 
 
-- Alternative
+Note that considerations of additional differential coverage rates by variables such as target screening population, age, and smoking status likely exist but are not considered in our simulation, which is a limitation of our model.
 
-.. todo::
+.. image:: screening_coverage_plot.png
 
-  Document target screening coverage rate and scale-up algorithm
+The above figure demonstrates the overall screening coverage rates that should be used in our model. Specifically, the baseline scenario population coverage rate should be 5.9% from 2020 to 2040. The alternative scenario population coverage rate should scale up from 5.9% in 2020 at a rate of one percent per year until 2035 and then remain constant at 20.9% from 2035 to 2040.
```

Comment:
Why did we pick 2035 here?  I ask because all the other models finish the scale-up by 2030.

Reply:
Oh, this is because the Japan scale up achieved 15% increase in 15 years, so I was modeling off of that. Do we want to adapt so it's consistent with the others? Either 15% in 10 years or 10% in 10 years?